src/apps/create-patron-user-info/UserInfo.tsx

```diff
@@ -76,17 +76,17 @@ const UserInfo: FC<UserInfoProps> = ({ cpr }) => {
             changePatron={changePatron}
             patron={patron}
           />
+          <PincodeSection required changePincode={setPin} />
           {t("createPatronChangePickupHeaderText") && (
-            <h2 className="text-body-small-regular mt-32 mb-16">
+            <h2 className="text-subtitle mt-32 mb-16">
```

Comment:
Consider showing these UI changes to the business sooner rather than later.

It seems as if they are outside the scope of the original change.

Reply:
They'll se it when they test it. We also talked about it in the last meeting.